deepchecks/tabular/checks/model_evaluation/simple_model_comparison.py

```diff
@@ -165,7 +165,8 @@ def run_logic(self, context: Context) -> CheckResult:
         # Multiclass have different return type from the scorer, list of score per class instead of single score
         if task_type in [TaskType.MULTICLASS, TaskType.BINARY]:
             n_samples = test_label.groupby(test_label).count()
-            classes = [clazz for clazz in test_dataset.classes if clazz in train_dataset.classes]
+            classes = [clazz for clazz in test_dataset.classes_in_label_col
```

Comment:
same here, why we don't care about all the classes?

Reply:
because we can't have scores for non existing classes